### ISSUE TITLE
Import Crash entire project, can't open project again

### ISSUE THREAD

**Issue by rieniter** [godotengine/godot#36057]
**Godot version:**
3.2
**OS/device including version:**
Windows 10
**Issue description:**
![2020-02-10_030212](https://user-images.githubusercontent.com/34160378/74108969-e87ca980-4bb1-11ea-9c2d-95be0deb89c1.png)
**Steps to reproduce:**
After try to change and set import default to some texture file .png
![2020-02-10_030753](https://user-images.githubusercontent.com/34160378/74109038-8cfeeb80-4bb2-11ea-8998-9b98ef491a90.png)
 at FileSystem Dock, i suddenly get crash and I can't open my project anymore, it was load into project about 1-2 second and then get crashed, I have to using video record to capture this error

**Comment by Calinou** [comment]
Can you reproduce this issue in any other project?

**Comment by rieniter** [comment]
> Can you reproduce this issue in any other project?

Yes, it running fine in other project, so i decide to remove my entire imported folder and it worked fine

**Comment by KoBeWi** [comment]
Corrupted/incompatible import files prevent project from opening. This can be easily observed when opening 3.x project in the 4.0-dev with Vulkan or vice versa. Maybe we could somehow force reimporting of broken files (the problem would be detecting this). This would make migrating to 4.0 less painful too.

**Comment by KoBeWi** [comment]
Doesn't seem to happen in 4.4.
Comment if can still reproduce.
